### PR TITLE
[WIP] @orta => Deploy with CocoaPods

### LINKED ISSUE #343: CocoaPods Keys is a deploy-blocker
You can test this easily by running Eidolon on a device. 

My _guess_ is that the root Pods project needs something changed to be able to host source code. I had a better idea about why a few days ago but those memories are lost. 

A quick fix can be made by using an alias to the `Eidolon*` files from inside the app source. 

A real fix will need to go in CP-Keys. But this can be enough to keep Eidolon moving.

### PR DESCRIPTION
**Pull request by ashfurrow**
Submitting work as I accomplish it. I'm doing some unrelated pair programming tomorrow, so it may be Tuesday before another commit. 

Gonna depend on https://github.com/orta/cocoapods-keys/pull/31 , too. 

Fixes #343.

### CODE REVIEW

**Comment by ashfurrow** [comment]
I should note that since I was able to build and run on a device after making manual changes to _only_ the Eidolon Keys stuff, I don't anticipate other, big problems deploying with CocoaPods through Hockey. If I do, I'll let @neonichu and @mrackwitz know.

**Comment by ashfurrow** [comment]
(This should fail, since it's running on the old version of cocoapods-keys.)

File: Podfile [hunk 1]
```diff
@@ -48,12 +50,12 @@ pod 'ARAnalytics/HockeyApp'
 
 pod 'CardFlight'
 pod 'ECPhoneNumberFormatter'
-pod 'UIImageViewAligned', :git => "https://github.com/orta/UIImageViewAligned.git"
+pod 'UIImageViewAligned', :git => "https://github.com/ashfurrow/UIImageViewAligned.git"
```
Comment (orta) [inline Podfile]: My PR for this got merged, and it's on trunk, not sure what you added but removing this may be doable

Comment (ashfurrow) [inline Podfile]: Only thing I changed was to add a tag on my fork ( https://github.com/ashfurrow/UIImageViewAligned/commit/6f8cb5669b219ef08c90c2e2e5a082cadfe7565b ) – if it's on trunk, then perfect :+1:

**Comment by orta** [comment]
looks good

File: Podfile (hunk 1 above)
Comment (orta) [inline Podfile]: its not, I mentioned it on my issue

Comment (ashfurrow) [inline Podfile]: See it – cool.

**Pull request merged**